functions/billing/main.py

```diff
@@ -42,26 +44,40 @@
 # [END functions_billing_limit]
 
 # [START functions_billing_slack]
-
 # See https://api.slack.com/docs/token-types#bot for more info
 BOT_ACCESS_TOKEN = 'xxxx-111111111111-abcdefghidklmnopq'
-
-CHANNEL_ID = 'C0XXXXXX'
+CHANNEL = 'C0XXXXXX'
 
 slack_client = slack.WebClient(token=BOT_ACCESS_TOKEN)
 
 
 def notify_slack(data, context):
     pubsub_message = data
 
-    notification_attrs = json.dumps(pubsub_message['attributes'])
-    notification_data = base64.b64decode(data['data']).decode('utf-8')
-    budget_notification_text = f'{notification_attrs}, {notification_data}'
+    # Default to blank strings to make testing the function easier
+    # See https://cloud.google.com/billing/docs/how-to/budgets-programmatic-notifications#notification_format
+    notification_attr = ''
```

Comment:
If this is _purely_ for testing, is there a way we can do without it?

(AFAIK we generally try to avoid any mention of test code in our samples, unless the test code itself has some associated educational/explanatory value.)

Reply:
Gotcha, let me rework that area.